### PR TITLE
Update slot_booking_bbmp.js

### PR DESCRIPTION
**Pull request by ymishra-se**
Added rerun slot search until slot available. Will show popup if slots available. Need to add some delay in rerun and desktop notification on success.

### CODE REVIEW

File: cypress/integration/slot_booking_bbmp.js [hunk 1]
```diff
@@ -2,45 +2,46 @@ describe("My First Test", () => {
   it("Open Cowin App!", () => {
     cy.viewport(1000, 1000);
     cy.visit("https://selfregistration.cowin.gov.in/");
-    cy.get("input#mat-input-0").type("<your_mobile_number>");
+    cy.get("input#mat-input-0").type("9968726089");
```
Comment (ojha-ravi) [inline cypress/integration/slot_booking_bbmp.js]: You can remove your Phone Number :) ... I am going to have the phone number through env variables

File: cypress/integration/slot_booking_bbmp.js [hunk 2]
```diff
@@ -2,45 +2,46 @@ describe("My First Test", () => {
...
+  * 4. Below method will rerun untill it will find a slot 
+  * 5. Need to add some delay 
+  */
+  it('.select() - select an option in a <select> element', 
+  {
+    "retries": 200000
+  },() => {    
+    // https://on.cypress.io/select   
+    
+    cy.get('.custom-checkbox').click('right')
```
Comment (ojha-ravi) [inline cypress/integration/slot_booking_bbmp.js]: This is really nice.
For some reason in consecutive attempts the test is failing here:
```
cy.get(".custom-checkbox").click("right");
``` 

So, I think we can do a something like this:
```
    it("Search by District", () => {
      cy.get(".custom-checkbox").click("right");
    });

    it("Select BBMP", () => {
      cy.get('[formcontrolname="state_id"]').click();
      cy.wait(100);
      cy.get('[role="listbox"]').contains("Karnataka").click();
      cy.wait(200);
      cy.get('[formcontrolname="district_id"]').click();
      cy.wait(100);
      cy.get('[role="listbox"]').contains("BBMP").click();
      cy.wait(200);
    });

   it(
      ".select() - select an option in a <select> element",
      {
        retries: 200000,
      },
      () => {
             ....
             ....
      });
)}
```

Thoughts??

**Review by ojha-ravi — APPROVED** [review]
This is nice, Thanks. Please do have a look at the comments.

File: cypress/integration/slot_booking_bbmp.js (hunk 2 above)
Comment (ymishra-se) [reply cypress/integration/slot_booking_bbmp.js]: yeah this makes more sense. we need to click on search & find slots in each run. One more thing it logoff after some runs so we need to run these retries only if url is appointment.

File: cypress/integration/slot_booking_bbmp.js (hunk 1 above)
Comment (ymishra-se) [reply cypress/integration/slot_booking_bbmp.js]: awesome :)

**Comment by ojha-ravi** [comment]
@ymishra-se If you update the PR, i will merge this. Thanks again.

**Review by ojha-ravi — APPROVED** [review]

**Pull request merged**